app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person in people:
+        person_list.append(Person(name=person["name"], age=person["age"]))
+
+    for person in people:
+        current_person = Person.people[person["name"]]
+
+        if "wife" in person and person["wife"]:
+            current_person.wife = Person.people[person["wife"]]
+
+        if "husband" in person and person["husband"]:
+            current_person.husband = Person.people[person["husband"]]
```

Comment:
Similar to the previous comment, this line assumes the husband's name exists in the `people` dictionary. Add error handling to manage cases where the name is not found.